sections/sprint-4/camera-handling.tex

```diff
@@ -0,0 +1,30 @@
+\section{Camera handling}
+This section will explain how the camera in the game is handled.
+Since the playing field can have a varying size the camera needs to be able to fit the whole playing field on the screen so that it can be rendered.
+Another problem is the usage of VR headsets that allow the player to turn the camera's view by looking in different directions.
+This causes the playing field to go out of the player's view which is a problem since they can then no longer see their position.
```

Comment:
```suggestion
This can cause the playing field to go out of the player's view, which is a problem since they can then no longer see their position.
```